packages/odyssey-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -15,9 +15,12 @@ import type { ReactElement, ComponentPropsWithRef } from "react";
 import { withTheme } from "@okta/odyssey-react-theme";
 import { useCx, useOid, useOmit } from "../../utils";
 import styles from "./Tooltip.module.scss";
-import { Text } from "../Text";
+import { Box } from "../Box";
 export interface TooltipProps
-  extends Omit<ComponentPropsWithRef<"aside">, "style" | "className" | "role"> {
+  extends Omit<
+    ComponentPropsWithRef<"aside">,
+    "style" | "className" | "role" | "color"
+  > {
```

Comment:
We should build a simple generic type that allows this type of repeated type definition to be defined in one place. DRY FTW

Going further, we should also add "color" to the omit list for omitProps so the type annotations and run time behavior match

Also consider that props that are available on an `aside` element, like `role` are not available here (as we hardcode it below). That is one more place we can improve the types

Reply:
I'm fine making a new card to address adding an additional type, but I do not believe there is much dryness here beyond style and className. All the other props (including color) are applied case by case.

Reply:
https://oktainc.atlassian.net/browse/OKTA-459590

Reply:
thanks @andrewberg-okta going to move that into our Feb sprint